CHANGELOG.md

```diff
@@ -4,17 +4,27 @@ Unreleased Changes
 2.1.0 (2019-02-14)
 ------------------
 
-* Feature - Aws::Rails::Mailer - Adds the Amazon SES message ID as a header to raw emails after sending, for tracking purposes. See [related GitHub pull request #25](https://github.com/aws/aws-sdk-rails/pull/25).
+* Feature - Aws::Rails::Mailer - Adds the Amazon SES message ID as a header to
+raw emails after sending, for tracking purposes. See
+[related GitHub pull request #25](https://github.com/aws/aws-sdk-rails/pull/25).
 
 2.0.1 (2017-10-03)
 ------------------
 
-* Issue - Ensure `aws-sdk-rails.initialize` executes before `load_config_initializers`
+* Issue - Ensure `aws-sdk-rails.initialize` executes before
+`load_config_initializers`
 
 2.0.0 (2017-08-29)
 ------------------
 
-* Upgrading - Support version 3 of the AWS SDK for Ruby. This is being released as major version 2 of `aws-sdk-rails`, though the APIs remain the same. Do note, however, that we've changed our SDK dependency to only depend on `aws-sdk-ses`. This means that if you were depending on other service clients transitively via `aws-sdk-rails`, you will need to add dependencies on the appropriate service gems when upgrading. Logger integration will work for other service gems you depend on, since it is wired up against `aws-sdk-core` which is included in the `aws-sdk-ses` dependency.
+* Upgrading - Support version 3 of the AWS SDK for Ruby. This is being released
+as major version 2 of `aws-sdk-rails`, though the APIs remain the same. Do note,
```

Comment:
Nitpick: setting the wrap margin to 72 characters (instead of 80) can make side-by-side diffs easier to read on screens with less real estate.